src/services/suggestionDiagnostics.ts

```diff
@@ -68,6 +69,9 @@ namespace ts {
                 }
             }
 
+            if (isFunctionLikeDeclaration(node) || isArrowFunction(node) || isMethodDeclaration(node)) {

```

Comment:
`isArrowFunction` and `isMethodDeclaration` are already subsumed by `isFunctionLikeDeclaration`. Just keep `isFunctionLikeDeclaration`